weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CallPage.tsx

```diff
@@ -33,7 +39,17 @@ export const CallPage: FC<{
   if (call.loading) {
     return <CenteredAnimatedLoader />;
   } else if (call.result === null) {
-    return <div>Call not found</div>;
+    return (
+      <>
+        <Button
+          icon="close"
+          variant="ghost"
+          onClick={() => close()}
+          style={{position: 'absolute', top: 0, right: 0, margin: '10px'}}
```

Comment:
The error panel is getting a scrollbar here - your browser is probably hiding it from you. I think you will need something more like:
```
      <div style={{display: 'flex', flexDirection: 'column', height: '100%'}}>
        <div style={{alignSelf: 'flex-end', margin: 10}}>
          <Button icon="close" variant="ghost" onClick={close} />
        </div>
        <div style={{flex: 1}}>
          <ErrorPanel title="Call not found" />
        </div>
      </div>
```

Reply:
Check out https://weightsandbiases.slack.com/archives/C02U6LLHP9S/p1709848285251229?thread_ts=1709832863.887669&cid=C02U6LLHP9S

Reply:
yikes, okay great callout, scrollbars are now `always` shown!